float32 vx
float32 vy
float32 vz
float32 yaw
string frame_id
bool auto_arm
---
bool success
string message
